homeassistant/components/rabbitair/__init__.py

```diff
@@ -0,0 +1,83 @@
+"""The Rabbit Air integration."""
+from __future__ import annotations
+
+import asyncio
+from datetime import timedelta
+import logging
+
+from rabbitair import Client, State, UdpClient
+
+from homeassistant.components import zeroconf
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_ACCESS_TOKEN,
+    CONF_HOST,
+    CONF_SCAN_INTERVAL,
+    Platform,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DEFAULT_SCAN_INTERVAL, DOMAIN, KEY_COORDINATOR, KEY_DEVICE
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS: list[Platform] = [Platform.FAN]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Rabbit Air from a config entry."""
+
+    hass.data.setdefault(DOMAIN, {})
+
+    host: str = entry.data[CONF_HOST]
+    token: str = entry.data[CONF_ACCESS_TOKEN]
+
+    _LOGGER.debug("Initializing with host %s, token %s", host, token[:5])
```

Comment:
The integration is already logged by Home Assistant and we should never log tokens :)

```suggestion
```